app/src/main/java/org/mozilla/focus/home/HomeFragment.kt

```diff
@@ -259,9 +266,27 @@ private fun onBindCustomHomeTile(uiLifecycleCancelJob: Job, holder: TileViewHold
         }
 
         // Wait for both to complete so we can animate them together.
-        iconView.setImageBitmap(screenshot.await()) // TODO: if null, provide placeholder.
+        val screenshotToRound = screenshot.await() // TODO: if null, provide placeholder.
+        val cornerRadius = 8.toFloat()
+        // Need copy of screenshot bitmap because it is immutable
+        val copyOfScreenshot = Bitmap.createBitmap(screenshotToRound.width, screenshotToRound.height, screenshotToRound.config)
+        val canvas = Canvas(copyOfScreenshot)
+        val paint = Paint()
+        // Need to set isAntiAlias to true because it smooths out the edges of what is being drawn
+        paint.isAntiAlias = true
+        // Need the shader to replicate the edge color if the shader draws outside of its original bounds
+        paint.shader = BitmapShader(screenshotToRound, Shader.TileMode.CLAMP, Shader.TileMode.CLAMP)
```

Comment:
I don't understand - when will the shader draw outside the original bounds?

I think we just need the BitmapShader in order to use the bitmap in the paint, which is how it gets in the canvas.